HelpLinks.md

```diff
@@ -0,0 +1,10 @@
+# Good Git and GitHub Links
+
+To learn about GitHub you should try the [https://guides.github.com/activities/hello-world/](hello-world exercise).
```

Comment:
The format is wrong here you should try using a different markdown format.  Try `[](url)`

Reply:
fixed!